### PR TITLE
Feat/sort new affiliate

### LINKED ISSUE #368: Check if user lose KYC
If the user loses KYC, the system should send him an email to let them know this authentication is needed to continue using the Proton Affiliate Program

### LINKED ISSUE #372: Pagination in New Affiliates does not display the right number
Screen Shot 2022-01-25 at 11 31 43 AM

### PR DESCRIPTION
**Pull request by leisterfrancisco**
### What does this PR do?

- Resolve #372 
- Resolve #368

### Steps to test

1. `make run`

### CODE REVIEW

File: hapi/src/services/join-request.service.js [hunk 1]
```diff
@@ -5,6 +5,12 @@ const { mailTemplate } = require('../utils/templates')
 const { hasuraUtil, eosUtil, mailUtil } = require('../utils')
 const affiliateService = require('./affiliate.service')
 
+const JOIN_REQUEST_STATUS_IDS = {
+  PENDING_KYC: 1,
+  PENDING_APPROVEMENT: 2,
```
Comment (xavier506) [inline hapi/src/services/join-request.service.js:10]: `PENDING_APPROVAL`

File: hapi/src/services/join-request.service.js [hunk 2]
```diff
@@ -49,33 +55,82 @@ const findByAccount = async account => {
...
+
+  const requesters = await findByStatus(JOIN_REQUEST_STATUS_IDS.PENDING_KYC)
+
+  for (const requester of requesters) {
+    const hasKYC = await affiliateService.checkKyc(requester.account)
+
+    if (!hasKYC) continue
+
+    await update(requester.account, {
+      status: JOIN_REQUEST_STATUS_IDS.PENDING_APPROVEMENT
```
Comment (xavier506) [inline hapi/src/services/join-request.service.js:121]: PENDING_APPROVAL

File: webapp/src/routes/Admin/index.js [hunk 3]
```diff
@@ -128,6 +128,60 @@ const headCellReferralPayment = [
...
+    label: 'PAID',
+    value: affiliateUtil.REFERRAL_STATUS_IDS.PAID
+  }
+]
+
+const joinRequestFilterValues = [
+  { label: 'allStatus' },
+  {
+    label: 'VERIFIED_KYC_VERIFICATION',
+    value: affiliateUtil.JOIN_REQUEST_STATUS_IDS.PENDING_APPROVEMENT
```
Comment (xavier506) [inline webapp/src/routes/Admin/index.js:165]: PENDING_APPROVAL

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by tetogomez — APPROVED** [review]
LGTM

**Pull request merged**